static/templates/authentication/register.html

```diff
@@ -74,8 +74,10 @@
         you agree to provide Daemo accurate and complete information about you and your business, and you authorize Daemo to share it and transaction
         information related to your use of the payment processing services provided by Stripe. Daemo does not store
         any banking or credit card information.</div-->
-    <div class="authentication-error" layout="row" ng-show="register.error && form.$pristine"
-         ng-bind="register.error"></div>
+    <div class="authentication-error" layout="row" ng-show="registrationEnabled == false && form.$pristine">
```

Comment:
registrationEnabled isn't in the scope and are you setting the form to $pristine if you are using it as a condition here?